### DISCUSSION TITLE
Failing to install Webcalendar-1.2.7. on strato.de

### DISCUSSION THREAD

**Discussion by xaver-ius** [craigk5n/webcalendar#445] [Q&A]
I copied all extracted files from the zip to a folder webcalendar of my homepage on strato.de
SSL is activated, PHP-Version is set to 8.2.
But if I type my url the only reaction is an error message: _getPhases: could not find '' in _filePhaseMap.
If I type my url with ..../install/index.php nothing happens at all
An old discussion to this error doesn't help me, because it is related to sending of reminders
It seems to be a very basic mistake - but help me please

**Comment by xaver-ius** [comment]
The Webcalendar-package offered from strato has version 1.9.1.
It uses  MariaDB 10.6.
The installation of this package is just a click and an admin-account 
**But it's not possible to add users!** 
A error message box with no error code is shown
All other functions seem to work
[Available PHP-versions are 8.0, 8.1. (recommended) and 8.2.] but any of them makes a difference

**Comment by xaver-ius** [comment]
Installing the offered package and accessing .../install/index.php directly, I got access to the installing script but it couldn't update to the newer DB-version by clicking the button and the script wasn't presented too.

**Comment by craigk5n** [comment]
Sorry but v1.2.7 did not support PHP8.  You'll need to get a more recent version.
